plugins/scaleway/api_key_test.go

```diff
@@ -0,0 +1,79 @@
+package scaleway
+
+import (
+	"testing"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/plugintest"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func TestAPIKeyImporter(t *testing.T) {
+	plugintest.TestImporter(t, APIKey().Importer, map[string]plugintest.ImportCase{
+		"Environment variables": {
+			Environment: map[string]string{
+				"SCW_ACCESS_KEY":              "SCWSYXTFI97NSEXAMPLE",
+				"SCW_SECRET_KEY":              "d9b67b48-873c-8ece-8270-e1e15example",
+				"SCW_DEFAULT_ORGANIZATION_ID": "11111111-2222-3333-4444-55555example",
```

Comment:
Let's include all envvars in this test.

Reply:
I see two tests for the importer now, only for a simple config and one for a full config. Seems like an okay approach to me, marking as resolved.